hugo/lib/getArticles.js

```diff
@@ -81,6 +81,7 @@ Promise.coroutine(function*() {
     });
   } catch (err) {
     console.error(err);
+    process.exit(6);
```

Comment:
Any reason you're choosing `6` here? If we're gonna differentiate between the various places in this script that it could error out and exit, then I think that makes sense to use different numbers at each `process.exit()`. And I'd be fine with us doing that here. Otherwise, I'd prefer just sticking with `process.exit(1)` for the sake of consistency.